static/app/components/modals/inviteMissingMembersModal/index.tsx

```diff
@@ -0,0 +1,327 @@
+import {Fragment, useState} from 'react';
+import styled from '@emotion/styled';
+
+import {closeModal, ModalRenderProps} from 'sentry/actionCreators/modal';
+import {Button} from 'sentry/components/button';
+import ButtonBar from 'sentry/components/buttonBar';
+import Checkbox from 'sentry/components/checkbox';
+import LoadingIndicator from 'sentry/components/loadingIndicator';
+import {
+  InviteModalHook,
+  InviteModalRenderFunc,
+  StatusMessage,
+} from 'sentry/components/modals/inviteMembersModal';
+import {InviteStatus} from 'sentry/components/modals/inviteMembersModal/types';
+import {MissingMemberInvite} from 'sentry/components/modals/inviteMissingMembersModal/types';
+import PanelItem from 'sentry/components/panels/panelItem';
+import PanelTable from 'sentry/components/panels/panelTable';
+import RoleSelectControl from 'sentry/components/roleSelectControl';
+import TeamSelector from 'sentry/components/teamSelector';
+import {Tooltip} from 'sentry/components/tooltip';
+import {IconCheckmark, IconCommit, IconGithub, IconInfo} from 'sentry/icons';
+import {t, tct, tn} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {MissingMember, Organization, OrgRole} from 'sentry/types';
+import {trackAnalytics} from 'sentry/utils/analytics';
+import {uniqueId} from 'sentry/utils/guid';
+import useApi from 'sentry/utils/useApi';
+import {
+  StyledExternalLink,
+  Subtitle,
+} from 'sentry/views/settings/organizationMembers/inviteBanner';
+
+export interface InviteMissingMembersModalProps extends ModalRenderProps {
+  invitableRoles: OrgRole[];
+  missingMembers: MissingMember[];
+  organization: Organization;
+}
+
+export function InviteMissingMembersModal({
+  missingMembers,
+  organization,
+  invitableRoles,
+}: InviteMissingMembersModalProps) {
+  const start = missingMembers?.map(member => ({
```

Comment:
nit: more descriptive variable name here, perhaps something like `initialMemberInvites`, likewise i feel that the state variable name should be 

`[memberInvites, setMemberInvites]`